Tests fail with git versions >= 2.9.1

This issue is fairly straightforward. I'll send a PR soon.
In git 2.9.1 the wording of the message from "git status" changed from "working directory" to "working tree"

See:
https://raw.githubusercontent.com/git/git/master/Documentation/RelNotes/2.9.1.txt
>  * "git status" used to say "working directory" when it meant "working tree".

The change was made in [commit 2a0e6cdeda](https://github.com/git/git/commit/2a0e6cdedab306eccbd297c051035c13d0266343) of the git repo.

There's a test in standard-version that tries to match the message from git status, and therefore gives a false-negative result to the test.
```
  17 passing (1m)
  1 failing

  1) cli CHANGELOG.md exists commits all staged files:
     AssertionError: expected 'On branch master\nnothing to commit, working tree clean\n' to match /On branch master\nnothing to commit, working directory clean\n/
      at Context.<anonymous> (test.js:112:21)
```

It can be fixed and made backwards compatible by adjusting the assertion regex from `working directory clean` to `working (directory|tree) clean`.

--- a/test.js
+++ b/test.js
@@ -109,7 +109,7 @@ describe('cli', function () {
       var content = fs.readFileSync('CHANGELOG.md', 'utf-8')
       var status = shell.exec('git status')
 
-      status.should.match(/On branch master\nnothing to commit, working directory clean\n/)
+      status.should.match(/On branch master\nnothing to commit, working (directory|tree) clean\n/)
       status.should.not.match(/STUFF.md/)
 
       content.should.match(/1\.0\.1/)

